src/routes_student.py

```diff
@@ -64,11 +64,11 @@ def student_get_assignment(netid, cid, aid):
 
 			user = db.get_user(netid)
 			commit = get_latest_commit(netid, user["access_token"], course["github_org"])
-
+			feedback_url = f'https://github-dev.cs.illinois.edu/{course["github_org"]}/{netid}/tree/{course["feedback_branch_name"]}' if "feedback_branch_name" in course else None
```

Comment:
The base url is hardcoded here. It will be better if you can extract it out to be a constant in the config file.

Reply:
If you are adding variables to config.py, don't forget to modify sample_config.py to reflect that.